spec/acceptance/approval_from_web_app.feature

```diff
@@ -1,106 +0,0 @@
-Feature: Approving a proposal from the web application
-  Scenario: An approver visits the page to approve
-    Given a proposal
-    And the proposal has an approval for 'supervisor1@test.gov' in position 1
-    Given the logged in user is 'supervisor1@test.gov'
-
-    When I go to the approval_response page without a token
-    And I should see 'Approve'
-    When I click 'Approve'
-    Then I should see alert text 'You have approved'
-
-  Scenario: An approver visits the page after previously responding
```

Comment:
this is covering the same use case as the token feature spec from the other spec file